open_discussions_api/client.py

```diff
@@ -3,6 +3,9 @@
 
 from open_discussions_api.users.client import UsersApi
 from open_discussions_api.utils import get_token
+from open_discussions_api.channels.client import ChannelsApi
+
+EXPIRATION_DELTA_SECONDS = 60 * 60
```

Comment:
This must've leaked through a rebase, it moved to `utils.py`.

Reply:
oh oops, indeed that is what happened